bin.js

```diff
@@ -3,16 +3,23 @@
 import sade from 'sade'
 import open from 'open'
 import { getPkg } from './lib.js'
-import { createSpace, registerSpace, createDelegation } from './index.js'
+import { createSpace, registerSpace, createDelegation, upload } from './index.js'
 
 const cli = sade('w3')
 
 cli
   .version(getPkg().version)
   .example('up path/to/files')
 
+cli.command('up <file>')
+  .alias('upload', 'put')
+  .describe('Store a file(s) to the service and register an upload.')
+  .option('--no-wrap', 'Don\'t wrap input files with a directory.')
+  .option('-H, --hidden', 'Include paths that start with ".".')
+  .action(upload)
+
 cli.command('open <cid>')
-  .describe('open CID on https://w3s.link')
+  .describe('Open CID on https://w3s.link')
```

Comment:
i been lowercasing all the things. we can go proper sentence case if you want it (grandad)